### ISSUE TITLE
Info / hover popup does not work

### ISSUE THREAD

**Issue by BlohoJo** [vitalikp/flagfox#1]
Tried on WinXP + Firefox 32.0.3 and Win7 x64 + Firefox x64 54.0.1, same results.

Instead of popup, a tiny blank box appears.

FlagFox ESR 5.2.4 (github, sha256 efae0d4a758c7e5ae495272c079b6fd12a0e2a59fb4452d7ac5a52495cb9e1c4)
![flagfox_nopopup](https://user-images.githubusercontent.com/8242464/35981997-a9eefac8-0cbc-11e8-8032-511baf32259a.png)

FlagFox 5.2.3 (addons.mozilla.org)
![flagfox_popup](https://user-images.githubusercontent.com/8242464/35982623-3f9b86e4-0cbe-11e8-8a75-117bc489d5d9.png)

Made sure to uninstall FlagFox 5.2.3 first. Tried restarting browser after uninstall and install, made no difference.

**Comment by vitalikp** [comment]
I can't reproduce this issue. (win7 x64 + Firefox x64 54.0.1).

Try looking errors in browser console Ctrl+Shift+J
(first clear all and then try reproduce this issue).

**Comment by BlohoJo** [comment]
In Win7 x64 Firefox x64 54.0.1, browser console shows:

`NS_ERROR_UNEXPECTED: Component returned failure code: 0x8000ffff (NS_ERROR_UNEXPECTED) [nsIStringBundle.GetStringFromName]  flagfox.jsm:854`


Relevant section of flagfox.jsm:
```
850        function addLabeledLine(labelID,lineValue)
851        {
852            var row = window.document.createElement("row");
853            var label = window.document.createElement("label");
854            label.setAttribute("value", strings.GetStringFromName(labelID));
855            label.setAttribute("style", "font-weight: bold;");
856            var value = window.document.createElement("label");
857            value.setAttribute("value", lineValue);
858            row.appendChild(label);
859            row.appendChild(value);
860            rows.appendChild(row);
861        }
```

**Comment by BlohoJo** [comment]
Is it possible, that I might be using another add-on that uses "labelID" ??

**Comment by vitalikp** [comment]
I think, problem with locale.

Try on empty profile.

**Comment by vitalikp** [comment]
/* disable extentions sign */
user_pref("xpinstall.signatures.required", false);

try this version.

[flagfox-5.2.5-dev.xpi.tar.gz](https://github.com/vitalikp/flagfox/files/1709414/flagfox-5.2.5-dev.xpi.tar.gz)

**Comment by BlohoJo** [comment]
That version fixes it! :)

**Comment by vitalikp** [comment]
Issue fixed by commit 881c514b906d65aaf4b7444278aba193435f2216

**Comment by BlohoJo** [comment]
[Release](https://github.com/vitalikp/flagfox/releases) is still at ESR 5.2.4 version, might want to update it to latest commit! :)
